pkg/controller/backup_controller.go

```diff
@@ -652,6 +656,37 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 				backupLog.Error(err)
 			}
 		}
+
+		//wait for all the VSBs to be complete
+		backupLog.Info("Fetching SnapMoverClient...")
+		volumeSnapMoverClient, err := GetVolumeSnapMoverClient()
+		if err != nil {
+			backupLog.Error(err)
+		}
+
+		//grab all the VSB from all the Namespaces included in the backup
+		backupLog.Info("Grabbing all the VSBs from all the Namespaces included in the backup...")
```

Comment:
I think this is short-sighted. What if a user uses label selector and doesn't set included namespaces?